runhouse/servers/obj_store.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, *args):
 def get_cluster_servlet(
     create_if_not_exists: bool = False,
     runtime_env: Optional[Dict] = None,
+    cluster_config: Optional[Dict] = None,
```

Comment:
dont love changing the usage of this method solely for the purposes of supporting the unit test .. can we just initialize the cluster servlet by calling ray.remote separately? (or put that speciifc code block in a helper function)

Reply:
Yeah, makes sense. 

Decided to go with the helper function approach, since we are initializing the NodeServlet and the Servlet in a very similar way, and this approach might be relevant to those initializations as well.

However, I currently created a helping function just for the ClusterServlet init, just to unblock this PR. Creating a generic helper function for all servlets might be a bit trickier (there are some nuances to take into consideration), so I think it will be wiser to do it in a separate PR. 

@jlewitt1 wdyt?

Reply:
cc: @carolineechen